javascript/javascript2/week1/index.js

```diff
@@ -0,0 +1,98 @@
+// 1.Find the shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function shortestWord(a) {
```

Comment:
It will be good practice to use some meaningful words instead of some letters for eg you can use 'words' instead of 'a'

Reply:
yes, you are right, here I think we only need one parameter, so I use one letter instead. it is meaningless.